### ISSUE TITLE
Building master on Windows never completes

### ISSUE THREAD

**Issue by mvysny** [vaadin/base-starter-gradle#3]
Running `gradlew vaadinClean clean build -Pvaadin.productionMode` with an empty pnpm cache on Windows 10 will never complete - the command will freeze forever.

Connecting VisualVM to the GradleDaemon process and dumping threads will reveal that the execution is blocked in the pnpm execution:

```
"Execution worker for ':'" #178 prio=5 os_prio=0 tid=0x0000000018b4b000 nid=0x3d7c runnable [0x0000000029bbb000]
   java.lang.Thread.State: RUNNABLE
        at java.lang.ProcessImpl.waitForInterruptibly(Native Method)
        at java.lang.ProcessImpl.waitFor(ProcessImpl.java:449)
        at com.vaadin.flow.server.frontend.TaskRunNpmInstall.runNpmInstall(TaskRunNpmInstall.java:374)
        at com.vaadin.flow.server.frontend.TaskRunNpmInstall.execute(TaskRunNpmInstall.java:118)
        at com.vaadin.flow.server.frontend.NodeTasks.execute(NodeTasks.java:634)
        at com.vaadin.gradle.VaadinBuildFrontendTask.runNodeUpdater(VaadinBuildFrontendTask.kt:132)
        at com.vaadin.gradle.VaadinBuildFrontendTask.vaadinBuildFrontend(VaadinBuildFrontendTask.kt:75)
```

My hypothesis is that pnpm is printing too much stuff into stdout which is not properly piped out. The buffer is rather small; when the buffer overflows, Windows will simply stop pnpm until the stdout buffer drains. Unfortunately there's nothing reading the stdout buffer and thus the buffer never drains.

I have no direct proof, but I have several items supporting this claim:

1. When a Maven project is built on the same drive, it pre-populates the `.pnpm-store` folder. This will cause the gradle build to succeed, most probably because pnpm is not doing much download and will thus not overflow the stdout buffer.
2. As opposed to Maven build, Gradle build prints nothing by default to stdout, not even when the `--info` switch is used.

## Steps to reproduce

1. Make sure that there is no `C:\.pnpm-store`, `D:\.pnpm-store` on any of your hard drives. This will force pnpm to redownload all dependencies, reproducinig the issue.
2. Run  `gradlew vaadinClean clean build -Pvaadin.productionMode` on this project on master.

**Comment by mvysny** [comment]
Reported to vaadin-gradle-plugin as https://github.com/vaadin/vaadin-gradle-plugin/issues/91

**Comment by mvysny** [comment]
This should be fixed, both in Gradle plugin 0.14.5.1 and 20.0.0.alpha6. Closing as fixed, please reopen if the issue persists.
